pkg/controllers/user/monitoring/metricExpression.go

```diff
@@ -2446,8 +2446,7 @@ metadata:
     metric: grpc-client-receive-bytes-sum-rate
     source: rancher-monitoring
 spec:
-  expression: sum(rate(etcd_network_client_grpc_received_bytes_total[5m]))  *
-    8 / 1024
+  expression: sum(rate(etcd_network_client_grpc_received_bytes_total[5m]))
   legendFormat: Client traffic in
```

Comment:
also need to update the graph unit

Reply:
Which unit wants to update? The `graph: rpc-client-traffic` are all using `bps` now.